### DISCUSSION TITLE
mingw-w64-clang-i686-jansson was removed?

### DISCUSSION THREAD

**Discussion by leleliu008** [msys2/MSYS2-packages#4923] [General]
I encountered `error: target not found: mingw-w64-clang-i686-jansson` when run `pacboy -S jansson:p` for `CLANG32` env

**Comment by Biswa96** [comment]
clang32 packages are being removed gradually https://www.msys2.org/news/#2024-09-23-starting-to-drop-the-clang32-environment

Please use 64 bit environment.

**Reply by leleliu008** [reply]
@Biswa96 Thank you.
